lsm/lsm.cpp

```diff
@@ -1,26 +1,93 @@
 #include "lsm.h"
+#include <iostream>
+#include <string>
+#include <variant>
+#include <vector>
+#include <fstream>
+#include <limits>
+#include <bitset>
+#include "MurmurHash3.h"
 
 void BloomFilter::insert_key(int32_t k) {
-  auto key_hash = std::hash<int32_t>{}(k);
-  auto last_d = key_hash % 10;
-  bset[last_d] = 1;
+    uint32_t seed = 1;
+    uint64_t key_hash;
```

Comment:
murmuhash3_x86_32 accepts a pointer to a 32-bit integer as its `out`, not a 64-bit integer.